pkg/application/options.go

```diff
@@ -243,8 +248,14 @@ func WithLoggerSettingsFromConfig(app *App) {
 		settings := &loggerSettings{}
 		config.UnmarshalKey("mon.logger", settings)
 
+		outputFile, err := os.OpenFile(settings.Output.File, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0644)
+		if err != nil {
+			return err
```

Comment:
Can we do something like `return fmt.Errorf("Failed to open logfile '%s': %w", settings.Output.File, err)`? I would avoid to have a failure of `error: permission denied` without knowing where this is coming from (not quite sure what os.OpenFile returns in case of an error, but I alread have seen it quite a lot to be told something does not exist for example without getting told what does not exist).

Reply:
definitely a good idea, thanks :)